### ISSUE TITLE
Netflow collection: Ntopng + Nprobe Protocol always Unkwnown and category Unspecified, ports are not guessed

### ISSUE THREAD

**Issue by SARBLA** [ntop/ntopng#6052]
Good day, team. saw it already on both Centos and Ubuntu.. Am i missing something. 
All flows that are received from nprobe have App Unknown and Category UNspecified. 

You mentioned on your page as description for your product 
Produce long-term reports for several network metrics including throughput and L7 application protocols
Top talkers (senders/receivers), top ASs, top L7 application protocols 

But what can be analysed if it is always Unknown, no clear port mapping for apps and categories or they not working

I fully undertand that we have no packets contenets in Netflow, in my case yes. 

Below example: 
Flow table
![Flow issue](https://user-images.githubusercontent.com/93220610/140516003-da6e1539-9b13-45ef-9088-6a6a9d78d74b.PNG)

Historical View table:
![Flow issue - historical](https://user-images.githubusercontent.com/93220610/140518787-42c63369-b708-4c77-b967-ac71787280f5.PNG)

**Comment by SARBLA** [comment]
Thanks to NTOPNGF support we found that issue was with incorrect export fields from Network devices.
flow record DC_V9
  match ipv4 source address
  match ipv4 destination address
  match ip protocol
  match ip tos
  match transport source-port
  match transport destination-port
  collect counter bytes
  collect counter bytes long <--- this was removed to fix an issue. 
  collect counter packets
 collect counter packets long  <<-- this was removed to fix an issue. 
collect timestamp sys-uptime first
  collect timestamp sys-uptime last
  collect ip version
